geth/peers/signal.go

```diff
@@ -0,0 +1,48 @@
+package peers
+
+import (
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/status-im/status-go/geth/signal"
+)
+
+const (
+	// DiscoveryStarted is sent when discv5 server was started.
```

Comment:
Possibly better name it as node here like in doc above?

Reply:
do you suggest to rename it as NodeStarted?

i think it must be discovery because it signals to a status react that we are actively looking for peers

Reply:
Oh, no, bad expressed, sorry. In your doc file you talked about "node", here about "server". I only stumbled about it and thought about harmonising in comment. Identifier is fine. 👍

Reply:
i will rephrase in the doc